src/formats/onenote.ts

```diff
@@ -492,21 +610,14 @@ export class OneNoteImporter extends FormatImporter {
 					codeElement.innerHTML = '```\n' + element.innerHTML + '\n```';
 				}
 				else {
-					element.remove();
 					// Append the content and add fences in case there's no next element
 					codeElement.innerHTML = codeElement.innerHTML.slice(0, -3) + element.innerHTML + '\n```';
 				}
 			}
 			else if (element.nodeName === 'BR' && inCodeBlock) {
 				codeElement.innerHTML = codeElement.innerHTML.slice(0, -3) + '\n```';
-				element.remove();
 			}
 			else {
-				if (inCodeBlock) {
```

Comment:
Why are these lines being removed?

Reply:
That part was responsible for fixing line breaks in code blocks, but now line breaks are fixed in the entire file, so it's no longer necessary.